app/components/article-media-thumbnail.js

```diff
@@ -41,12 +45,12 @@ export default Component.extend(
 		hasFigcaption: computed.or('caption', 'showTitle'),
 
 		showTitle: computed('type', function () {
-			return this.get('type') === 'video' && this.get('title');
+			return this.get('type') === 'video' || this.get('isOgg') && this.get('title');
```

Comment:
@Wikia/x-wing this should be `(this.get('type') === 'video' || this.get('isOgg')) && this.get('title')`